--- makefile	2015-01-13 12:56:59.000000000 -0500
+++ makefile.cuda	2015-01-13 12:57:02.000000000 -0500
@@ -7,12 +7,7 @@
 siftgpu_enable_opencl = 0
 #------------------------------------------------------------------------------------------------
 # enable CUDA-based SiftGPU?
-simple_find_cuda = $(shell locate libcudart.so)
-ifneq ($(simple_find_cuda), )
- 	siftgpu_enable_cuda = 1
-else
-	siftgpu_enable_cuda = 0
-endif
+siftgpu_enable_cuda = 1
 
 CUDA_INSTALL_PATH = /usr/local/cuda
 #change  additional  settings, like SM version here if it is not 1.0 (eg. -arch sm_13 for GTX280)
